versioned_docs/version-2.0/how_to_guides/monitoring/rules.mdx

```diff
@@ -55,6 +55,8 @@ There are four actions you can take with an automation rule:
 - **Add to annotation queue**: Add the trace to an annotation queue.
 - **Run online evaluation**: Run an online evaluation on the trace. For more information on online evaluations, you can refer to [this guide](./online_evaluations).
 - **Trigger webhook**: Trigger a webhook with the trace data. For more information on webhooks, you can refer to [this guide](./webhooks).
+- **Extend data retention**: If you are using base [data retention](../../concepts/usage_and_billing/data_retention_billing), allows you
```

Comment:
Should we add something to note that all the others will still extend it, but this one is just exclusively used if not action is needed?

Reply:
@akira updated - ptal